src/components/deliverables/Deliverables.vue

```diff
@@ -14,7 +14,9 @@
                 <button v-bind:disabled="!canBagFinalSelection" class=" btn btn-primary marg-bot-4" data-toggle="modal" data-target=".bd-example-modal-lg">Bag Final Results</button>
                   <div>
                     <h2 class="sub-title">Item: <span>{{item.itemName}}</span></h2>
+                    <h2 class="sub-title">Item ID: <span>{{item.itemId}}</span></h2>
```

Comment:
This is referring to the externalId, right? itemId would usually refer to the PrimareyKey Id. Can you change itemId to externalId, to be consistent and avoid confusion.